CHANGELOG.md

```
@@ -1,3 +1,15 @@
+# [0.6.0-beta.1](https://github.com/mi-examples/cs-helper/compare/v0.5.1...v0.6.0-beta.1) (2026-02-24)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)